### PR TITLE
Nick fixes

### PR DESCRIPTION
**Pull request by njohnso6**
This fixes issues #191 and #173 
The merge conflict that would result will be switching "shell" to "run" in the Snakefile

### CODE REVIEW

File: docs/getting-started.rst [hunk 1]
```diff
@@ -67,16 +67,20 @@ variable, so that any executables there are found first. It does not affect any
...
+**It is recommended that you create a separate environment directory for each 
+project and include it with the project**.
+That way you can update packages in each project independently of
+any others, and yet the environment will always be close at hand. This is 
+especially good practice in shared space as others can easily find and activate 
+the environment specific to the project.
+Here we use the name "env" for the new environment to clearly denote its contents, 
+but you can use anything. Note that here we specify the channels to use, which 
+include ``bioconda`` which depends on ``conda-forge``, and ``lcdb`` which provides 
+the ``lcdblib`` package used by these workflows.
```
Comment (daler) [inline docs/getting-started.rst]: This is much improved!

File: workflows/rnaseq/config/rnaseq_patterns.yaml [hunk 2]
```diff
@@ -11,9 +11,9 @@ libsizes:
   bam:     'data/rnaseq_samples/{sample}/{sample}.cutadapt.bam.libsize'
 fastq_screen: 'data/rnaseq_samples/{sample}/{sample}.cutadapt.screen.txt'
 featurecounts:
-  s0: 'data/rnaseq_samples/{sample}/{sample}.cutadapt.bam.featurecounts.s0.txt'
-  s1: 'data/rnaseq_samples/{sample}/{sample}.cutadapt.bam.featurecounts.s1.txt'
-  s2: 'data/rnaseq_samples/{sample}/{sample}.cutadapt.bam.featurecounts.s2.txt'
+  s0: 'data/rnaseq_samples/combined/cutadapt.bam.featurecounts.s0.txt'
```
Comment (daler) [inline workflows/rnaseq/config/rnaseq_patterns.yaml]: There is an existing `rnaseq_aggregation` directory, created for e.g. the multiqc rule (see [this pattern](https://github.com/lcdb/lcdb-wf/blob/master/workflows/rnaseq/config/rnaseq_patterns.yaml#L24)). I think these new aggregated counts would be better living there.

**Comment by daler** [comment]
Thanks. I'm pretty sure tests are currently failing because the rnaseq.Rmd is written to expect the individual featureCounts runs rather than a single file containing all of them, so we'll need to fix that before merging.

File: workflows/rnaseq/config/rnaseq_patterns.yaml (hunk 2 above)
Comment (daler) [reply workflows/rnaseq/config/rnaseq_patterns.yaml]: (fixed in e867fd7)

**Comment by njohnso6** [comment]
Just to make sure it's recorded, I added some further changes to this branch even though not all the tests pass yet.

**Comment by daler** [comment]
This is looking good -- merging into v1.5rc!

**Pull request merged**